src/ports/catalog/component.ts

```diff
@@ -91,13 +96,15 @@ export function createCatalogComponent(options: {
         catalogItems = enhanceItemsWithPicksStats(catalogItems, picksStats)
       }
     } catch (e) {
+      console.error(e)
```

Comment:
This `console.error` is on purpose, right?

Reply:
yup, so it gets logged and we can debug what's the query erroring about